pkg/karmadactl/exec.go

```diff
@@ -64,12 +52,15 @@ func NewCmdExec(karmadaConfig KarmadaConfig, parentCommand string) *cobra.Comman
 	o.GlobalCommandOptions.AddFlags(cmd.Flags())
 
 	cmdutil.AddPodRunningTimeoutFlag(cmd, defaultPodExecTimeout)
-	cmdutil.AddJsonFilenameFlag(cmd.Flags(), &o.FilenameOptions.Filenames, "to use to exec into the resource")
+	cmdutil.AddJsonFilenameFlag(cmd.Flags(), &o.KubectlExecOptions.FilenameOptions.Filenames, "to use to exec into the resource")
+	// TODO support UID
```

Comment:
What the TODO means?
Can you give the details? and maybe other guys can work on it.

Reply:
It's copied from `kubectl exec`, which has been introduced since 7 years old. 

Unfortunately, I cannot find more details about it. I' ll remove it in this pr.